frontend/src/metabase/visualizations/visualizations/PieChart/utils.ts

```diff
@@ -0,0 +1,23 @@
+export function getMaxLabelDimension(
+  d3Arc: d3.svg.Arc<d3.svg.arc.Arc>,
+  slice: d3.svg.arc.Arc,
+  index = 0,
```

Comment:
Are you planning to use this parameter later? But I don't see any usage of this 3rd parameter yet.